README.md

```diff
@@ -36,3 +36,11 @@ yarn
 yarn dev
 yarn test
 ```
+
+### Lighthouse
+
+`yarn lighthouse` will test the **UAT URL** for the project, so you will need to configure that in `lighthouserc.json`.
+
+`yarn lighthouse:local` will test the **BUILD** of the project, so if there is no build present you will need to run `yarn build` initially.
+
+This is done so that local testing will reflect UAT as closely as possible, with the build process having minified the JS, CSS, etc...
```

Comment:
might make more sense for `yarn lighthouse` to default to local, and have `yarn lighthouse:remote` for the remote version

Reply:
👍